lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -5847,5 +5848,14 @@ DefPrimitive('\DeclareUnicodeCharacter Expanded {}', sub {
 # LaTeX now includes textcomp by default.
 RequirePackage('textcomp');
 
+#%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+# Expl3 "Experimental LaTeX 3" is no longer Experimental!
+# It is beginning to be built into latex.ltx
+# We WILL need a new strategy to keep up; probably based in some form
+# of pre-read/pre-processed latex.ltx !
```

Comment:
Maybe once they stabilize it fully we can make ourselves a new `Latex3.pool` file and forget about it for a little bit...